scripts/mixins/thermalinnovation.zs

```diff
@@ -0,0 +1,55 @@
+#loader mixin
+
+#mixin Mixin
+#{targets: "cofh.thermalinnovation.item.ItemDrill"}
+zenClass MixinItemDrill {
```

Comment:
Cant see mining level changes ingame. They are still default.


https://github.com/user-attachments/assets/e2cc11af-80fc-46c0-accf-f7c59594e613

Reply:
Latest changes wont help. Drills still have default mining levels: 

![](https://i.imgur.com/XwCjzLn.png)

Reply:
Seems like issue is actually fixed.